AGENTS.md

```diff
@@ -0,0 +1,69 @@
+# Metal3 Website - AI Agent Instructions
+
+Instructions for AI coding agents. For content guidelines, see [GUIDELINES.md](GUIDELINES.md).
+
+## Overview
+
+Public-facing website for Metal3 at <https://metal3.io>. Built with
+Jekyll static site generator, hosted on GitHub Pages.
+
+## Repository Structure
+
+| Directory | Purpose |
+|-----------|---------|
+| `_posts/` | Blog posts (dated markdown files) |
+| `_layouts/` | Jekyll page templates |
+| `_includes/` | Reusable HTML components |
+| `_faqs/` | FAQ entries |
+| `assets/` | Images, CSS, JS |
+| `hack/` | CI scripts (markdownlint, spellcheck, shellcheck) |
+
+## Testing Standards
+
+CI uses GitHub Actions. Run locally before PRs:
+
+| Command | Purpose |
+|---------|---------|
+| `make lint` | Run all linters |
+| `make serve` | Serve locally on port 4000 |
+| `./hack/markdownlint.sh` | Markdown linting |
+| `./hack/spellcheck.sh` | Spell checking (cspell) |
```

Comment:
We should include pre-commit here.

Reply:
Done, also removed duplicated hack scripts, as make lint includes both.